homeassistant/components/niko_home_control/light.py

```diff
@@ -22,7 +22,7 @@
 MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=1)
 SCAN_INTERVAL = timedelta(seconds=30)
 
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({vol.Required(CONF_HOST): cv.string})
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({vol.Optional(CONF_HOST): cv.string})
```

Comment:
I think this falls under ADR-0007 and ADR-0010.
Changing the schema is not allowed, and the integration would need to be migrated to config-flow.

See https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md and https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md